cabal.GHC-8.8.4.config

```diff
@@ -93,6 +93,7 @@ constraints: any.Cabal ==3.0.1.0,
              any.cryptohash-md5 ==0.11.100.1,
              any.cryptohash-sha1 ==0.11.100.1,
              cryptol +relocatable -static,
+             cryptol-saw-core +build-css,
```

Comment:
Is this a necessary change?

Reply:
I would guess that this is an artifact of multiple merges...? I have no idea what it is